### DISCUSSION TITLE
TIPO DI LEZIONE

### DISCUSSION THREAD

**Discussion by sempliciniluigi89** [iisgiua/giuaschool#356] [Idee e proposte]
Sarebbe possibile mettere una volta firmato il registro una tendina con la scelta del tipo di lezione se in PRESENZA,MODALITA' MISTA,DDI,SINCRONA ,ASINCRONA stessa cosa per gli alunni se partecipano da casa etc?

**Comment by trinko** [comment]
Visto che la DAD non è più prevista, mi pare poco utile.
